interactive_engine/executor/store/global_query/src/store_impl/groot/global_graph.rs

```diff
@@ -75,14 +75,13 @@ impl GlobalGraph {
         }
     }
 
-    fn parse_condition(
-        _condition: Option<&Condition>,
-    ) -> Option<&maxgraph_store::db::api::condition::Condition> {
-        None
-    }
-
-    fn parse_property_id(_prop_ids: Option<&Vec<PropId>>) -> Option<&Vec<PropertyId>> {
-        None
+    fn parse_property_id(prop_ids: Option<&Vec<PropId>>) -> Option<Vec<PropertyId>> {
```

Comment:
I think we can unify `PropId` and `PropertyId`, as a type of `u32`. Otherwise, this would be confusing.

Reply:
how about do this in a seprate `pr`.